dl/src/main/scala/com/intel/analytics/bigdl/dataset/Utils.scala

```diff
@@ -28,9 +30,12 @@ object Utils {
       require(totalBatch % (nodeNumber * coreNumber) == 0
         , s"total batch size($totalBatch) can't be divided by node number($nodeNumber) * " +
           s"core number($coreNumber), please change your batch size")
-      require(totalBatch >= nodeNumber * coreNumber * 2
-        , s"total batch size($totalBatch) should be at least two times of node number" +
-          s"($nodeNumber) * core number($coreNumber), please change your batch size")
+
+      if (totalBatch < nodeNumber * coreNumber * 2) {
+        logger.info(s"Warning: for better property, " +
+          s"total batch size($totalBatch) should be at least two times of node number" +
+          s"($nodeNumber) * core number($coreNumber), please optimize your batch size")
+      }
```

Comment:
Change to "... for better training speed, ... is recommended to be at least ... please tune your batch size accordingly"